internal/civo/bootstrapSecrets.go

```diff
@@ -143,7 +145,7 @@ func BootstrapCivoMgmtCluster(dryRun bool, kubeconfigPath string, gitProvider st
 		},
 		// git runner
 		{
-			ObjectMeta: metav1.ObjectMeta{Name: "controller-manager", Namespace: fmt.Sprintf("%s-runner", gitProvider)},
+			ObjectMeta: metav1.ObjectMeta{Name: gitRunnerSecretName, Namespace: fmt.Sprintf("%s-runner", gitProvider)},
```

Comment:
```suggestion
			ObjectMeta: metav1.ObjectMeta{Name: gitRunnerSecretName, Namespace: fmt.Sprintf("%s-runner", gitProvider)},
```
i think we need accommodate namespace to be `controller-manager` for github runner. maybe add another variable for each of the switches?